src/data-workspace/data-entry-cell/validation-tooltip.js

```diff
@@ -25,7 +25,10 @@ const TooltipManager = React.forwardRef(function TooltipManager(
         else {
             close()
         }
-    }, [invalid, active, open, close])
+        // onMouseOut, onMouseOver are not stable references
```

Comment:
We should fix that in our UI library. Could you create a Jira ticket?

Reply:
Birk has created one: https://dhis2.atlassian.net/browse/LIBS-359